### ISSUE TITLE
AttributeError: 'API' object has no attribute 'list_direct_messages'

### ISSUE THREAD

**Issue by gasull** [micahflee/semiphemeral#64]
```python
$ semiphemeral delete
[...]
Fetching direct message metadata for the last 30 days
Traceback (most recent call last):
  File "/usr/bin/semiphemeral", line 11, in <module>
    load_entry_point('semiphemeral==0.6', 'console_scripts', 'semiphemeral')()
  File "/usr/lib/python3/dist-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/usr/lib/python3/dist-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/usr/lib/python3/dist-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/lib/python3/dist-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/lib/python3/dist-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/usr/lib/python3/dist-packages/semiphemeral/__init__.py", line 70, in delete
    t.delete()
  File "/usr/lib/python3/dist-packages/semiphemeral/twitter.py", line 430, in delete
    for page in tweepy.Cursor(self.api.list_direct_messages).pages():
AttributeError: 'API' object has no attribute 'list_direct_messages'
```

**Comment by gasull** [comment]
I've looked into the code to fix this issue.  This could be fixed inserting a `try-except` on [line 430 of twitter.py](https://github.com/micahflee/semiphemeral/blob/797bd03cd16a067717fd066cbd03842929542d99/semiphemeral/twitter.py#L430).  But I don't know what message could be shown to the user.

What's the user supposed to do in this situation?

**Comment by kees** [comment]
I'm not seeing this error. What versions of the libraries do you have installed?

**Comment by gasull** [comment]
```
$ pipenv graph
bandit==1.6.2
  - GitPython [required: >=1.0.1, installed: 3.1.11]
    - gitdb [required: >=4.0.1,<5, installed: 4.0.5]
      - smmap [required: <4,>=3.0.1, installed: 3.0.4]
  - PyYAML [required: >=3.13, installed: 5.3.1]
  - six [required: >=1.10.0, installed: 1.15.0]
  - stevedore [required: >=1.20.0, installed: 3.2.2]
    - pbr [required: !=2.1.0,>=2.0.0, installed: 5.5.1]
black==20.8b1
  - appdirs [required: Any, installed: 1.4.4]
  - click [required: >=7.1.2, installed: 7.1.2]
  - mypy-extensions [required: >=0.4.3, installed: 0.4.3]
  - pathspec [required: <1,>=0.6, installed: 0.8.1]
  - regex [required: >=2020.1.8, installed: 2020.11.13]
  - toml [required: >=0.10.1, installed: 0.10.2]
  - typed-ast [required: >=1.4.0, installed: 1.4.1]
  - typing-extensions [required: >=3.7.4, installed: 3.7.4.3]
bpython==0.20.1
  - curtsies [required: >=0.3.0, installed: 0.3.4]
    - blessings [required: >=1.5, installed: 1.7]
      - six [required: Any, installed: 1.15.0]
    - wcwidth [required: >=0.1.4, installed: 0.2.5]
  - greenlet [required: Any, installed: 0.4.17]
  - pygments [required: Any, installed: 2.7.2]
  - requests [required: Any, installed: 2.25.0]
    - certifi [required: >=2017.4.17, installed: 2020.11.8]
    - chardet [required: >=3.0.2,<4, installed: 3.0.4]
    - idna [required: <3,>=2.5, installed: 2.10]
    - urllib3 [required: >=1.21.1,<1.27, installed: 1.26.2]
  - six [required: >=1.5, installed: 1.15.0]
  - wcwidth [required: Any, installed: 0.2.5]
Flask==1.1.2
  - click [required: >=5.1, installed: 7.1.2]
  - itsdangerous [required: >=0.24, installed: 2.0.0a1]
  - Jinja2 [required: >=2.10.1, installed: 3.0.0a1]
    - MarkupSafe [required: >=1.1, installed: 2.0.0a1]
  - Werkzeug [required: >=0.15, installed: 1.0.1]
pkg-resources==0.0.0
pudb==2019.2
  - pygments [required: >=1.0, installed: 2.7.2]
  - urwid [required: >=1.1.1, installed: 2.1.2]
pydocstyle==5.1.1
  - snowballstemmer [required: Any, installed: 2.0.0]
pylint==2.6.0
  - astroid [required: <=2.5,>=2.4.0, installed: 2.4.2]
    - lazy-object-proxy [required: ==1.4.*, installed: 1.4.3]
    - six [required: ~=1.12, installed: 1.15.0]
    - wrapt [required: ~=1.11, installed: 1.12.1]
  - isort [required: <6,>=4.2.5, installed: 5.6.4]
  - mccabe [required: <0.7,>=0.6, installed: 0.6.1]
  - toml [required: >=0.7.1, installed: 0.10.2]
PySocks==1.7.1
rope==0.18.0
SQLAlchemy==1.4.0b1
  - greenlet [required: Any, installed: 0.4.17]
tweepy==3.9.0
  - requests [required: >=2.11.1, installed: 2.25.0]
    - certifi [required: >=2017.4.17, installed: 2020.11.8]
    - chardet [required: >=3.0.2,<4, installed: 3.0.4]
    - idna [required: <3,>=2.5, installed: 2.10]
    - urllib3 [required: >=1.21.1,<1.27, installed: 1.26.2]
  - requests-oauthlib [required: >=0.7.0, installed: 1.3.0]
    - oauthlib [required: >=3.0.0, installed: 3.1.0]
    - requests [required: >=2.0.0, installed: 2.25.0]
      - certifi [required: >=2017.4.17, installed: 2020.11.8]
      - chardet [required: >=3.0.2,<4, installed: 3.0.4]
      - idna [required: <3,>=2.5, installed: 2.10]
      - urllib3 [required: >=1.21.1,<1.27, installed: 1.26.2]
  - six [required: >=1.10.0, installed: 1.15.0]
twine==3.2.0
  - colorama [required: >=0.4.3, installed: 0.4.4]
  - keyring [required: >=15.1, installed: 21.5.0]
    - jeepney [required: >=0.4.2, installed: 0.5.0]
    - SecretStorage [required: >=3.2, installed: 3.2.0]
      - cryptography [required: >=2.0, installed: 3.2.1]
        - cffi [required: >=1.8,!=1.11.3, installed: 1.14.3]
          - pycparser [required: Any, installed: 2.20]
        - six [required: >=1.4.1, installed: 1.15.0]
      - jeepney [required: >=0.4.2, installed: 0.5.0]
  - pkginfo [required: >=1.4.2, installed: 1.6.1]
  - readme-renderer [required: >=21.0, installed: 28.0]
    - bleach [required: >=2.1.0, installed: 3.2.1]
      - packaging [required: Any, installed: 20.4]
        - pyparsing [required: >=2.0.2, installed: 3.0.0b1]
        - six [required: Any, installed: 1.15.0]
      - six [required: >=1.9.0, installed: 1.15.0]
      - webencodings [required: Any, installed: 0.5.1]
    - docutils [required: >=0.13.1, installed: 0.16]
    - Pygments [required: >=2.5.1, installed: 2.7.2]
    - six [required: Any, installed: 1.15.0]
  - requests [required: >=2.20, installed: 2.25.0]
    - certifi [required: >=2017.4.17, installed: 2020.11.8]
    - chardet [required: >=3.0.2,<4, installed: 3.0.4]
    - idna [required: <3,>=2.5, installed: 2.10]
    - urllib3 [required: >=1.21.1,<1.27, installed: 1.26.2]
  - requests-toolbelt [required: !=0.9.0,>=0.8.0, installed: 0.9.1]
    - requests [required: >=2.0.1,<3.0.0, installed: 2.25.0]
      - certifi [required: >=2017.4.17, installed: 2020.11.8]
      - chardet [required: >=3.0.2,<4, installed: 3.0.4]
      - idna [required: <3,>=2.5, installed: 2.10]
      - urllib3 [required: >=1.21.1,<1.27, installed: 1.26.2]
  - rfc3986 [required: >=1.4.0, installed: 1.4.0]
  - setuptools [required: >=0.7.0, installed: 44.0.0]
  - tqdm [required: >=4.14, installed: 4.52.0]

```

**Comment by kees** [comment]
This should be solved with PR #89 now.
